frontend/src/scenes/session-recordings/player/inspector/PlayerInspectorControls.tsx

```diff
@@ -38,6 +38,33 @@ function TabButtons({
     const { tab, tabsState } = useValues(inspectorLogic)
     const { setTab } = useActions(inspectorLogic)
 
+    return (
+        <LemonTabs
+            size="small"
+            activeKey={tab}
+            onChange={(tabId) => setTab(tabId)}
+            tabs={tabs.map((tabId) => {
+                const TabIcon = TabToIcon[tabId]
+
+                return {
+                    key: tabId,
+                    label: (
+                        <div className="flex items-center gap-1">
+                            {TabIcon ? (
+                                tabsState[tabId] === 'loading' ? (
+                                    <Spinner textColored />
+                                ) : (
+                                    <TabIcon />
+                                )
+                            ) : undefined}
+                            <span>{capitalizeFirstLetter(tabId)}</span>
+                        </div>
+                    ),
+                }
+            })}
+        />
+    )
+
     return (
```

Comment:
unreachable code?

Reply:
Yep, don't know how that got past the linter